src/transaction.js

```diff
@@ -1221,21 +1222,24 @@ Transaction.prototype.summary = function() {
     initialSubmitIndex: this.initialSubmitIndex,
     lastLedgerSequence: this.lastLedgerSequence,
     state: this.state,
-    server: this._server ? this._server._opts.url : undefined,
```

Comment:
Why did we remove this? Cause there can be more than one server per transaction?